sn_protocol/src/lib.rs

```diff
@@ -121,7 +121,7 @@ impl NetworkAddress {
     // /// This implements the following equivalence:
     // ///
     // /// `self xor other = distance <==> other = self xor distance`
-    // pub fn for_distance(&self, d: Distance) -> libp2p::kad::kbucket::KeyBytes {
+    // pub fn for_distance(&self, d:  ) -> libp2p::kad::kbucket::KeyBytes {
```

Comment:
don't get the point why remove this type declare

Reply:
unintentional! was due to refactor before libp2p Distance was in, I suspect. Will re-add